docs/data-sources/service_principal.md

```diff
@@ -23,7 +23,7 @@ data "databricks_service_principal" "spn" {
 
 resource "databricks_group_member" "my_member_a" {
   group_id  = data.databricks_group.admins.id
-  member_id = data.databricks_service_principal.spn.id
```

Comment:
really, it would be easier to fix service principal data source to expose `id` instead of `sp_id` - this will match attributes of other resources.